controllers/local_db_secret.go

```diff
@@ -57,16 +55,12 @@ func (r *BackstageReconciler) handlePsqlSecret(ctx context.Context, statefulSet
 			return nil, fmt.Errorf("failed to get secret for PostgreSQL DB (%q), reason: %s", secretName, err)
 		}
 		// Create a secret with a random value
-		val := func(length int) string {
-			bytes := make([]byte, length)
-			if _, randErr := rand.Read(bytes); randErr != nil {
-				// Do not fail, but use a fallback value
-				return _defaultPostGresSecretValue
-			}
-			return base64.StdEncoding.EncodeToString(bytes) // Encode the password to prevent special characters
-		}(24)
-		sec.StringData["POSTGRES_PASSWORD"] = val
-		sec.StringData["POSTGRESQL_ADMIN_PASSWORD"] = val
+		pwd, pwdErr := generatePassword(24)
```

Comment:
I guess there are no needs in a function throwing error,
e g consider using "k8s.io/apimachinery/pkg/util/rand" 
like: rand.String(24)

Reply:
That makes sense to use it. I didn't know this function - thanks!

Reply:
Done in [`bbab5ab` (#152)](https://github.com/janus-idp/operator/pull/152/commits/bbab5ab1c94b85202e57e40b3d570969e36d64c4)

Reply:
can `k8s.io/apimachinery/pkg/util/rand` be used for cryptographic purposes?  FYI `crypto/rand` is recommended here: https://cheatsheetseries.owasp.org/cheatsheets/Cryptographic_Storage_Cheat_Sheet.html

Reply:
looks like it's not...k8s uses math rand which is unsafe

Reply:
> looks like it's not...k8s uses math rand which is unsafe

So you're suggesting to revert to `crypto/rand`?

Reply:
Okay - reverted to the previous implementation using `crypto/rand` in [`abf657e` (#152)](https://github.com/janus-idp/operator/pull/152/commits/abf657e49bc153c1649bf2f648657cac856231c3)